.github/workflows/publish-stack-set-to-service-catalog.yaml

```diff
@@ -66,18 +83,14 @@ jobs:
           working_dir: 'cdk.out'
           cdk_stack: DynatraceMonitoringRoleStack
```

Comment:
Should be DynatraceActivegateStack

Reply:
Also L114

Reply:
@melvabout is this the case? As you made these changes previously... Happy to change just let me know :)

Reply:
Name of the workflow, publish-stack-set-to-service-catalog.yaml doesn't reflect, what this is being used for anymore, based on as is I was going to strip it out as part of https://govukverify.atlassian.net/browse/OBS-87.  If you want to change it to DynatraceActivegateStack here and I will address DynatraceMonitoringRoleStack, DynatraceSSMProdStack and DynatraceSSMNonProdStack either in this workflow or alternates as part of OBS-87